### DISCUSSION TITLE
是否可以做一个包托管网站，方便社区分享和协作

### DISCUSSION THREAD

**Discussion by vkensou** [ejoy/ant#59] [Ideas]
既然引擎已经有了包的机制，那么有一个包的托管网站会很方便社区分享和协作。

**Comment by cloudwu** [comment]
我觉得用 github 就可以。因为 vfs 允许你把包放在不同路径下。

现在没有精力维护这样的网站。

**Reply by vkensou** [reply]
[openupm](https://openupm.com/)  这样的自助式网站就很方便。似乎前端界比较喜欢包管理和包托管网站。游戏引擎界很少有做这样功能的，目前发现就unity支持。某些引擎有简单的资产商店，但和包管理差太远了。
